content/doc/developer/views/symbols.adoc

```diff
@@ -0,0 +1,55 @@
+---
+title: Symbols
+layout: developerguide
+---
+
+pass:[<span class="version">Available since Jenkins 2.TODO.</span>]
+
+image::/images/developer/views/symbols.svg[Selection of Symbols]
+
+View the complete list of Symbols available to use in Jenkins on link:https://github.com/jenkinsci/jenkins/tree/master/war/src/main/resources/images/symbols[GitHub].
+
+[.lead]
+Jenkins Symbols are an extensive and consistent collection of icons that you can use on Jenkins or in your plugin.

```

Comment:
This sounds like a tip and is not really binding. Can we make it more clear that we want to use the same icon set for all plugin symbols if possible? I think only in cases where no other icon is available we should use other ones. Our goal should be to use the same icons throughout the whole application.

Reply:
That sounds good 👍 I've updated the page, hope that makes things clearer